pysb/importers/bngl.py

```diff
@@ -317,7 +317,9 @@ def _parse_rules(self):
             rule.rate_reverse = rev_rate
 
     def _parse_expressions(self):
-        expr_namespace = {p.name: p.value for p in self.model.parameters}
+        expr_namespace = {p.name: p for p in self.model.parameters}
```

Comment:
This can be drastically simplified since `ComponentSet` is dict-like and we aren't taking the `.value` of the parameters anymore:
```python
expr_namespace = dict(
    self.model.parameters | self.model.expressions_constant()
    | self.model.observables
)
```
(Some of that simplification could have been done previously, too, I guess) Depending on what's using it downstream, you probably don't even need to turn the combined ComponentSet into a dict.